### PR TITLE
Issue 6108

### PR DESCRIPTION
**Pull request by markbond1007**
## Description

#6108 -fixing the missing Google API key in the jjwg_Maps.php code

## Motivation and Context

Stops the over quota message in properly configured Google accounts

## How To Test This

Using the sample data, geocode the data once (should partially run if it fails), clear the data and run again, you should NOT get any missed data (assuming the addresses are all entered).

## Types of changes

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x ] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x ] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by horus68** [comment]
Try to make your PR against hotfix-7.8
Note: your branch need to be updated against the branch you want to make the pull request, so there will not be no confliting files or non related changed files.

**Comment by Dillon-Brown** [comment]
@markbond1007 I fixed the branch for you, would you be able to sign the clahub? you should see it right below the comment. Thanks.

**Comment by markbond1007** [comment]
No Problem - All signed for you!

File: modules/jjwg_Maps/jjwg_Maps.php [hunk 1]
```diff
@@ -893,11 +893,12 @@ function getGoogleMapsGeocode($address, $return_full_array = false, $allow_appro
         if (!(strpos($base_url, '?') > 0)) $base_url .= '?';
         // Add Address Parameter
         $request_url = $base_url . "&address=" . urlencode($address);
+        $request_url.="&key=".urlencode($this->settings['google_maps_api_key']);
         // Add Hash Parameter as MD5 of Concatenation of Address and Secret
-        if (!empty($this->settings['geocoding_api_secret'])) {
+      /*  if (!empty($this->settings['geocoding_api_secret'])) {
```
Comment (daniel-samson) [inline modules/jjwg_Maps/jjwg_Maps.php]: Please can we remove instead of commenting out code?

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**